recipe/cudatoolkit-dev-post-install.py

```diff
@@ -169,121 +127,76 @@ class LinuxExtractor(Extractor):
 
     def extract(self):
         print("Extracting on Linux")
-        runfile = os.path.join(self.src_dir, self.cu_blob)
+        runfile = self.blob_dir / self.cu_blob
         os.chmod(runfile, 0o777)
-        cmd = [
-            runfile,
-            "--silent",
-            "--toolkit",
-            "--toolkitpath",
-            str(self.src_dir),
-            "--override",
-        ]
-        try:
-            subprocess.check_call(cmd)
-        except subprocess.CalledProcessError as e:
-            print(
-                "ERROR: Couldn't install Cudatoolkit: \
-                   {reason}".format(
-                    reason=e
-                )
-            )
 
-    def cleanup(self):
-        blob_path = os.path.join(self.src_dir, self.cu_blob)
-        if os.path.exists(blob_path):
-            os.remove(blob_path)
-
-        else:
-            pass
+        with tempdir() as tmpdir:
+            cmd = [runfile, f"--extract={tmpdir}", f"--defaultroot={tmpdir}"]
```

Comment:
I think we'll need `--override` option to resolve the following error when extracting:
```
[INFO]: Setting extractPath=/tmp/tmpam64w639
[INFO]: Setting defaultpath=/tmp/tmpam64w639
[INFO]: Checking compiler version...
[INFO]: gcc location: which: no gcc in ((null))

[ERROR]: Missing gcc. gcc is required to continue.

```